sphinx/builders/latex/__init__.py

```diff
@@ -505,7 +505,7 @@ def validate_latex_theme_options(app: Sphinx, config: Config) -> None:
 
 def install_pakcages_for_ja(app: Sphinx) -> None:
```

Comment:
```suggestion
def install_packages_for_ja(app: Sphinx) -> None:
```
Maybe this can be included as well in this PR.

Reply:
Thank you for comment. I'll fix this in next PR.